client/src/test/scala/dwolla/cloudflare/AccessControlRuleClientSpec.scala

```diff
@@ -0,0 +1,346 @@
+package dwolla.cloudflare
+
+import cats.effect.Sync
+import com.dwolla.cloudflare._
+import com.dwolla.cloudflare.domain.model.Exceptions.UnexpectedCloudflareErrorException
+import com.dwolla.cloudflare.domain.model.{AccountId, AccountIdTag}
+import com.dwolla.cloudflare.domain.model.accesscontrolrules._
+import io.circe.Json
+import org.http4s.Status
+import org.http4s.client.Client
+import org.specs2.mutable.Specification
+import org.specs2.specification.Scope
+import io.circe.literal._
+
+class AccessControlRuleClientSpec extends Specification {
+
+  trait Setup extends Scope {
+    val accountId: AccountId = shapeless.tag[AccountIdTag][String]("account-id")
+
+    val authorization = CloudflareAuthorization("email", "key")
+    val fakeService = new FakeCloudflareService(authorization)
+  }
+
+  "list" should {
+    "list challenge rules" in new Setup {
+      val http4sClient = fakeService.client(fakeService.listAccessRules(Map(1 → SampleResponses.Successes.listAccessRulesPage1, 2 → SampleResponses.Successes.listAccessRulesPage2, 3 → SampleResponses.Successes.listAccessRulesPage3), accountId))
+      val client = buildAccessControlRuleClient(http4sClient, authorization)
+
+      val output: List[Rule] = client
+        .list(accountId, Option("challenge"))
+        .compile
+        .toList
+        .unsafeRunSync()
+      output must contain(
+        {
```

Comment:
It's not a big deal, but these `{}` are redundant here. The `Rule` instance can be passed directly to `contain`.